movement-migration/framework-upgrades/scripts/feature-flag-reconfig.move

```diff
@@ -14,8 +14,7 @@ script {
             67, // ConcurrentFungibleBalance
             40, // VMBinaryFormat7
             74, // EnumTypes
-            80, // NativeMemoryOperation
-            223, // new GGP
+            223, // STAKE_REWARD_USING_TREASURY
```

Comment:
can you keep 223 for GGP and use 224 for STAKE_REWARD_USING_TREASURY. also you deleted 80 by accident?